include/ginkgo/core/matrix/csr.hpp

```diff
@@ -1014,6 +1038,22 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         strategy_->process(row_ptrs_, &srow_);
     }
 
+    /**
+     * @copydoc scale(const LinOp *)
+     *
+     * @note  Other implementations of dense should override this function
+     *        instead of scale(const LinOp *alpha).
```

Comment:
copypasta :)
```suggestion
     * @note  Other implementations of Csr should override this function
     *        instead of scale(const LinOp *alpha).
```